plugins/postcss-gamut-mapping/README.md

```diff
@@ -0,0 +1,76 @@
+# PostCSS Gamut Mapping [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS Logo" width="90" height="90" align="right">][PostCSS]
+
+[<img alt="npm version" src="https://img.shields.io/npm/v/@csstools/postcss-gamut-mapping.svg" height="20">][npm-url] [<img alt="CSS Standard Status" src="https://cssdb.org/images/badges/gamut-mapping.svg" height="20">][css-url] [<img alt="Build Status" src="https://github.com/csstools/postcss-plugins/workflows/test/badge.svg" height="20">][cli-url] [<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+```bash
+npm install @csstools/postcss-gamut-mapping --save-dev
+```
+
+[PostCSS Gamut Mapping] lets you use wide gamut colors with gamut mapping for specific displays following the [CSS Color 4 Specification].
```

Comment:
Wondering if we should explain why this is important? Something in this line:

> This plugin ensures that colors get progressively enhanced at the closest gamut that the device supports.

By looking at the examples (unless you really know what's going on) it's somewhat difficult to understand why this would be useful. Thoughts?

Reply:
Good point!

I've added a bit more of an explainer to the docs.
I hope this is clear?

Reply:
https://github.com/csstools/postcss-plugins/blob/1e3febcf007058aaba75d77987a62792f34f1cfd/plugins/postcss-gamut-mapping/README.md

Reply:
This looks lovely!